src/draw_instanced.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
```

Comment:
Is this a port from the OpenGL Samples Pack?  If so, let's include a link to the original example in this file.